tsconfig.json

```diff
@@ -17,8 +17,9 @@
     "esModuleInterop": true,
     "module": "esnext",
     "jsx": "react",
-    "types": ["jest", "node"]
+    "types": ["jest", "node"],
+    "skipLibCheck": true
```

Comment:
Is this safe?

Reply:
Well, I had to do that not because of performance, but because service worker context is incompatible with normal browser context. And since it was checking the content of all the workbox libraries as if they were executed in browser context it was failing.
Apparently, this doesn't stop checking the codebase types against the libraries types, it just stop checking the libraries themselves internally.
Ideally we would need to have 2 typescript configs for the 2 different contexts, but I didn't manage to have that working.

Reply:
I see, didn't think about the context issue. I guess not checking the library definitions we create is fairly low risk anyway